lisa/testsuite.py

```diff
@@ -165,7 +201,7 @@ def set_status(
             self.status = new_status
             if new_status == TestStatus.RUNNING:
                 self._timer = create_timer()
-            self._send_result_message()
+            self._send_result_message(stacktrace)
```

Comment:
Can it just set into the test result, instead of pass as a parameter? It's no different situation like `message`.